src/redux/thunks/applyResource.ts

```diff
@@ -53,43 +52,18 @@ function applyK8sResource(
 function applyKustomization(
   resource: K8sResource,
   fileMap: FileMapType,
-  kubeconfig: string,
   context: string,
-  kustomizeCommand: KustomizeCommandType,
+  projectConfig: ProjectConfig,
```

Comment:
do we need the entire `projectConfig` here? 
I'm asking because using a selector to get the entire `projectConfig` in a component will make that component re-render each time any property from the config has changed. And from what I saw it's selected in a couple different components

Reply:
this method also uses projectConfig.kubeConfig?.path so it seemed better to pass the config instead of individual args